<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="pagelayoutwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>480</width>
     <height>560</height>
    </rect>
   </property>
   <widget class="QWidget" name="start_page">
    <widget class="QPushButton" name="continue_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>221</width>
       <height>170</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="clockdisplay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>480</width>
       <height>100</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="sensordisplay_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>230</width>
       <height>100</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLabel" name="clockdisplay_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>49</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Clock</string>
     </property>
    </widget>
    <widget class="QLabel" name="sensordisplay_1_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>sensor 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>221</width>
       <height>170</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="sensordisplay_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>230</width>
       <height>100</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sensordisplay_1_label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>sensor 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainmenu_page">
    <widget class="QPushButton" name="systeminfo_page_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>System Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editsettings_page_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainmenu_back_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manualcontrols_page_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Manual Controls</string>
     </property>
    </widget>
    <widget class="QPushButton" name="monitordata_page_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Monitor Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="monitordata_page">
    <widget class="QPushButton" name="monitordata_back_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="manualcontrols_page">
    <widget class="QPushButton" name="waternow_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Water Now</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lightswitch_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Light Switch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fanon_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fan On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="takepicture_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Take Picture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recorddata_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Record Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manualcontrols_back_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="systeminfo_page">
    <widget class="QPushButton" name="updatefirmware_page_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Update Firmware</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logexport_page_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Log Export</string>
     </property>
    </widget>
    <widget class="QPushButton" name="systeminfo_back_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="systemversion_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;&lt;System Version&gt;&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="editsettings_page">
    <widget class="QPushButton" name="irrigation_page_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Irrigation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editsettings_back_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="datetime_page_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date/Time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cameratoggle_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Camera Toggle</string>
     </property>
    </widget>
    <widget class="QSlider" name="brightness_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>25</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="brightness_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;&lt;Brightness&gt;&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="datetime_page">
    <widget class="QPushButton" name="datetime_back_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>470</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plus_btn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="datetimenumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>381</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>61</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="irrigation_page">
    <widget class="QPushButton" name="humidset_page_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Humid Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="irrigation_back_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="humidset_page">
    <widget class="QOpenGLWidget" name="openGLWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="humidset_back_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="statusbar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: gray
</string>
   </property>
   <widget class="QLabel" name="statusbar_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;Status Bar&gt;&gt; </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
